templates/misc/message.part.html

```diff
@@ -242,13 +243,17 @@
                             {% with profile_user=user|profile %}
                                 {% if profile_user.show_sign %}
                                     <div class="signature">
-                                        {{ profile.sign|emarkdown_inline }}
+                                        {% cache 300 message profile.sign %}
```

Comment:
Pourquoi as-tu mis "message"? `cache 300 profile.sign` semble suffisant, non?

Reply:
J'avais effectivement essayé avec seulement `profile.sign` mais ça ne marchait pas, la même signature était affichée sur tous les messages (c'était celle du premier). Je n'ai pas compris pourquoi, peut-être est-ce lié à l'utilisation du template dans des boucles `for` ?

Reply:
non il faut que tu dises que tu mets en cache la signature en fonction du profile. Donc un truc genre `{% cache 300 profile.user.username profile.sign %}`

Reply:
Ah d'accord, merci bien !

Cependant, j'ai du mal à comprendre pourquoi ça ne marche pas avec `profile.sign`. Le markdown de la signature n'est-il pas lui aussi unique pour chaque membre ? Du coup, vu qu'un utilisateur a toujours la même signature, pourquoi faut-il mettre aussi en cache selon la signature (vu qu'il y a une contrainte d'unicité sur le nom d'utilisateur) ?
(C'est juste par curiosité, t'es pas obligé de répondre. ;) )